content/en/docs/concepts/cluster-administration/api-priority-and-fairness.md

```diff
@@ -0,0 +1,125 @@
+---
+title: API priority and fairness
+content_template: templates/concept
+weight: 20
+---
+
+
+{{% capture overview %}}
+
+{{< feature-state state="alpha" >}}
+
+The control plane of Kubernetes is itself a resource (broadly
+construed) to which access is managed.  In particular, each apiserver
+applies some prioritization and fairness in its handling of requests.
+This includes not only kube-apiservers but aggregated apiservers too.
+Prioritization and fairness applies only to requests that are not
+_long running_.  Long running requests are WATCHes and EXECUTEs.
+
+{{% /capture %}}
+
+
+{{% capture body %}}
+
+## Status
+
+This feature is named "RequestManagement" and is alpha in
+release 1.16.  As an alpha feature, it is disabled by default.  When
+enabled it replaces a request filter that enforces simple aggregate
+concurrency limits with this feature, which can be seen as a more
+general version of that earlier functionality.
+
+## Outline
+
+Each apiserver independently applies prioritization and fairness,
+based on API objects that configure this feature.  This involves two
+concepts: priority level and flow schema.
```

Comment:
So does this mean that two requests to a load-balancer-fronted Kubernetes API endpoint could display different rate-limiting behaviour depending on which kube-apiserver backend the user's request happened to get routed to? That seems like a recipe for confusion. Or are we expecting that for HA k8s API setup we should be handling rate-limiting and balancing at the load-balancer or top-level proxy level?